packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -255,6 +257,7 @@ export const FUTURE_FLAGS: { [key: string]: boolean } = {
   [RDS_LOWERCASE_DB_IDENTIFIER]: true,
   [EFS_DEFAULT_ENCRYPTION_AT_REST]: true,
   [LAMBDA_RECOGNIZE_VERSION_PROPS]: true,
+  [LAMBDA_RECOGNIZE_LAYER_VERSION]: true,
```

Comment:
This line has been causing me problems. When set to `false`, the integ runner works fine. But as it is set to `true`, it seems like the integ runner picks this flag up and then changes the behavior for many of the lambda integ tests. 

A cursory glance at the integ runner shows me that future flags are a part of it:

https://github.com/aws/aws-cdk/blob/144b38ddde8994aa7c7b8fd292cb383c3b57fa9c/packages/%40aws-cdk/integ-runner/lib/runner/runner-base.ts#L369-L387

I see two paths forward -- either the integ runner shouldn't be picking up these flags, or we will have to be okay with a couple of lambda integ tests being changed as a result of this PR. I am leaning towards the latter being the path we want to go down, but want to get confirmation on it first.

Reply:
Agreed.